src/libstd/sys/common/stack.rs

```diff
@@ -345,4 +350,10 @@ pub unsafe fn get_sp_limit() -> uint {
     unsafe fn target_get_sp_limit() -> uint {
         1024
     }
+
+    #[cfg(target_os = "openbsd")] #[inline(always)]
+    unsafe fn target_get_sp_limit() -> uint {
+        // segmented stack is disabled
+        1024
+    }
```

Comment:
Could this actually go aheand and get consolidated with the above missing implementations as well? No need to define the same block 3 times :)